README.md

```diff
@@ -11,14 +11,14 @@ following clients and servers:
 ### Servers
 
 - Connect, using [Connect's Go implementation][connect-go]
-- gRPC, using [grpc-go][grpc-go]
+- gRPC, using [gRPC-go][grpc-go]
```

Comment:
I'd might just leave them as `grpc-go` and `grpc-web`, given that the names in repos are ["gRPC-Go"](https://github.com/grpc/grpc-go#grpc-go) and ["gRPC Web"](https://github.com/grpc/grpc-web#grpc-web--). Up to you, though.

Reply:
I don't have a preference on this, so I will just revert that part